attack/scripts/bashrc

```diff
@@ -227,37 +227,31 @@ welcome() {
   info ' '
   info "There are ${BOLD}${#MASTER_NODES[@]}${NORMAL} master and ${BOLD}${#WORKER_NODES[@]}${NORMAL} nodes in the cluster."
   info ' '
-  info "If you want to begin to work on the scenario from its starting point:"
+  if [[ -f /challenge.txt ]]; then
+    envsubst </challenge.txt
+  fi
+  info ' '
+  info 'To start the first task:'
+  info ' '
+  info "  \$ start_task 1"
+  info ' '
+  info "Begin to work on the scenario from its starting point:"
   info ' '
   info "  \$ starting_point"
   info ' '
-  info "Additionally, we have setup helpers to SSH to the master(s) and node(s)."
+  info 'Getting stuck? See the next hint or recap'
   info ' '
-  info "To SSH to a master:"
-  info "  \$ ssh_master 0"
+  info "  \$ next_hint"
+  info "  \$ show_hints"
   info ' '
-  info "To SSH to a node:"
-  info "  \$ ssh_node 1"
+  info "SSH to master(s), node(s), and a bastion:"
```

Comment:
Not the bastion, you're already in the attack continer on the bastion.. It's an internal instance not in the cluster but with network access to the cluster